zebrad/tests/acceptance.rs

```diff
@@ -351,3 +351,61 @@ fn valid_generated_config(command: &str, expected_output: &str) -> Result<()> {
 
     Ok(())
 }
+
+#[test]
+fn listener_address_test() -> Result<()> {
```

Comment:
We can run these tests in parallel, if we change the ports to 0.

Reply:
Changed all ports to 0 in https://github.com/ZcashFoundation/zebra/pull/951/commits/beb7370a3134bcfd4fd4540ecda1f3dda64e6adb